dbscripts/xml/upgrade.xml

```diff
@@ -97,6 +97,10 @@
 		<migration class="PKP\migration\upgrade\v3_4_0\I7249_UpdateUsersUniqueIndex"/>
 	</upgrade>
 
+	<upgrade minversion="3.3.0.0" maxversion="3.4.9.9">
+		<migration class="PKP\migration\upgrade\v3_5_0\I10962_UpdateEmailTemplateVariables"/>
+	</upgrade>
```

Comment:
here we any installation upgrading from 
1. `3.3.0-0` to `3.5.0-x`
2. `3.4.0-x` to `3.5.0-x` 

will have this migration applied . So will it work if we move it under the block of `<upgrade minversion="3.1.0.0" maxversion="3.4.9.9">` ?